geomesa-spark/geomesa_pyspark/src/main/python/.gitignore

```diff
@@ -0,0 +1,3 @@
+dist/
```

Comment:
some of the stuff is localized being our first python module, can be moved if we want to embrace...